site/docs/guide/payments.md

```diff
@@ -0,0 +1,3 @@
+---
+prev: ./deployment-types.md
```

Comment:
Awesome changes. Please write the rest.

Reply:
> Awesome changes. Please write the rest.

Yeap, xD) I'm passing the last exams tomorrow and plan to write some text from my notes.